### PR TITLE
schema done

### PR DESCRIPTION
**Pull request by vmaineng**

### CODE REVIEW

File: server/models/Skillset.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const SkillSetSchema = new Schema({
+  skillSet: {
+    type: [String],
+    required: [true, "Please choose a skillset"],
+    unique: false,
+    enum: ["Programming Languages", "Frameworks", "Other Technologies"],
```
Comment (hjkmines) [inline server/models/Skillset.js]: @vmaineng  Let's change this to all say without the "s" at the end

File: server/models/Skillset.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      "Rust",
+      "Kotlin",
+      "Swift",
+      "TypeScript",
+      "PHP",
+    ],
+  },
+  frameworkCategory: {
+    type: [String],
+    unique: true,
```
Comment (hjkmines) [inline server/models/Skillset.js]: let's change this to `false`

File: server/models/Skillset.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      "Ruby",
+      "Rust",
+      "Kotlin",
+      "Swift",
+      "TypeScript",
+      "PHP",
+    ],
+  },
+  frameworkCategory: {
+    type: [String],
```
Comment (hjkmines) [inline server/models/Skillset.js]: Let's have this as just `String`

File: server/models/Skillset.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+const SkillSetSchema = new Schema({
+  skillSet: {
+    type: [String],
+    required: [true, "Please choose a skillset"],
+    unique: false,
+    enum: ["Programming Languages", "Frameworks", "Other Technologies"],
+  },
+  programmingCategory: {
+    type: [String],
```
Comment (hjkmines) [inline server/models/Skillset.js]: Let's have this as just `String`

File: server/models/Skillset.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const SkillSetSchema = new Schema({
+  skillSet: {
+    type: [String],
```
Comment (hjkmines) [inline server/models/Skillset.js]: Let's have this as just `String`

File: server/models/Skillset.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+      "Angular",
+      "Vue",
+      "jQuery",
+      "Express.js",
+      "Node.js",
+      "Ruby on Rails",
+    ],
+  },
+  otherTechnologies: {
+    type: [String],
```
Comment (hjkmines) [inline server/models/Skillset.js]: Let's have this as just `String`

File: server/models/Skillset.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+const SkillSetSchema = new Schema({
+  skillSet: {
+    type: [String],
+    required: [true, "Please choose a skillset"],
+    unique: false,
+    enum: ["Programming Languages", "Frameworks", "Other Technologies"],
+  },
+  programmingCategory: {
+    type: [String],
+    unique: true,
```
Comment (hjkmines) [inline server/models/Skillset.js]: let's change this to `false`

File: server/models/Skillset.js [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+      "Vue",
+      "jQuery",
+      "Express.js",
+      "Node.js",
+      "Ruby on Rails",
+    ],
+  },
+  otherTechnologies: {
+    type: [String],
+    unique: true,
```
Comment (hjkmines) [inline server/models/Skillset.js]: let's change this to `false`

**Review by hjkmines — CHANGES_REQUESTED** [review]

**Review by hjkmines — APPROVED** [review]
nice job!

**Review by FunkyColdMedyna — APPROVED** [review]
LGTM !

**Review by vinhpeters — APPROVED** [review]
Looks good! The first programming language I had to learn was MATLAB, glad to see it did not make the cut.

**Pull request merged**